queue/abstract.lua

```diff
@@ -570,15 +570,44 @@ function method.start()
     end
 
     for _, tube_tuple in _queue:pairs() do
-        local tube = recreate_tube(tube_tuple)
-        -- gh-66: release all taken tasks on start
-        tube_release_all_tasks(tube)
+        -- Recreate tubes for registered drivers only.
+        -- Check if a driver exists for this type of tube.
+        if queue.driver[tube_tuple[4]] ~= nil then
```

Comment:
How about emitting a warning in the case?

Reply:
I think it will look like garbage when starting the queue instance registering the custom driver after box.cfg (). And will cause unnecessary questions from users. But if you insist, I won't mind.

Reply:
Your point is right (however it depends on how the log entry will be formulated). OTOH, silent no-op may be confusing if a driver registration does not occurs for some reason. I have no strict preference here, so I don't insist.